postprocessing/science/spectral_corner_frequency_from_surface_xdmf.py

```diff
@@ -243,7 +244,15 @@ def ApproximateEventDurationAndHypocenter(faultxdmf, faultxyz):
         ASl = LoadSingleComponentParallel("ASl", surfaceData=False)
     ASR = np.abs(np.gradient(ASl, dtFault, axis=1))
     slippingElement = np.where(np.amax(ASR, axis=0) > slipRateThreshold, 1, 0)
-    duration = slippingElement[np.argmax(slippingElement):-np.argmax(np.flip(slippingElement))].size * dtFault
+    if np.all(slippingElement == 1):
+        print("Warning: at least one element is constantly slipping")
```

Comment:
you mean 'there is always one element slipping during the whole simulation'?
(not necessarily always the same?).

Reply:
Yes, `"Warning: at least one element is always slipping"` is probably better phrased